tcl/Util.tcl

```diff
@@ -207,6 +207,28 @@ proc sta_warn_error { msg_id warn_error msg } {
   }
 }
 
+define_cmd_args "suppress_msg" {msg_ids}
+
+proc suppress_msg { args } {
+  parse_key_args "suppress_msg" args keys {} flags {}
```

Comment:
no need for this if there are no keywork or flag args

Reply:
Removed